### DISCUSSION TITLE
Session1 ZKP for DLOG

### DISCUSSION THREAD

**Discussion by xiegudong45** [zkp-co-learning/zkp-co-learn#20] [Q&A（提问题在隔壁~）]
想问下这道题里面b是如何计算的？维基百科中只提到b 是由g,y,h通过哈希计算出的一个0/1值，那具体这个hash函数应该怎么定义呢？
![image](https://user-images.githubusercontent.com/30063330/220659024-b600e6c7-e417-4e7e-b64f-b3d0f90fa1a0.png)

**Comment by lkiversonlk** [comment]
哪个b，s = (r + bx)(mod (p − 1)) 里这个？
